### PR TITLE
fix test_acl_sharenetworks from components

### PR DESCRIPTION
**Pull request by DaanHoogland**
### Description

This PR... has some problems. It contains a good enhancement of the cloudstackTestCase, but a lot of tests depend on it not being there. creating a new PR with only the revert.
see #7033

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Enhancement (improves an existing feature and functionality)
- [x] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### CODE REVIEW

**Comment by weizhouapache** [comment]
@blueorangutan package

**Comment by weizhouapache** [comment]
@DaanHoogland there is a lint check error

File: tools/marvin/marvin/cloudstackTestCase.py [hunk 1]
```diff
@@ -99,7 +99,9 @@ def cleanup_resources(cls, api_client, resources):
             Delete resources (created during tests)
         """
         for obj in resources:
-            if isinstance(obj, VirtualMachine):
+            if isinstance(obj, Domain):
+                obj.delete(api_client, cleanup=true)
```
Comment (DaanHoogland) [inline tools/marvin/marvin/cloudstackTestCase.py:103]:
```suggestion
                obj.delete(api_client, cleanup=True)
```

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
Too many other tests are suffering from this solution. closing.

**Pull request closed without merging**